list/to_c_or_not_to_c.md

```diff
@@ -30,102 +37,144 @@ The Super Game Boy adds a few minor things, such as a customizable screen border
 
+* Purists prefer to also run their games on hardware, which is possible thanks to flashcarts. My personal recommendation is [krikzz's carts](http://krikzz.com/store/), particularly the [Everdrive GB X5](https://krikzz.com/store/home/47-everdrive-gb.html).
+
+Side note : if you are using VBA or VBA-rr, **stop using them right now**. These emulators are extremely inaccurate, and also contain **severe security flaws**. I strongly urge you to ditch these emulators and spread the word.
 
-# Summary
 
-If your question is "*What should I use for my game project ?*", then you're in the right section.
 
-The first question you should ask yourself is what languages you know. If you have never worked with ASM, C or C++, you should start by learning C. This will introduce to how you interface with the hardware when you're close to it (pointers, above all)., If you grasped C's concepts (most importantly pointers), give ASM a go. Even if you don't manage to get working ASM code, it actually helps a lot (especially on such a constrained system) to know what's "under the hood". Especially with GBDK's crappy lib, it's particularly useful to use ASM-level debugging tools to figure what exactly is wrong.
 
-Now, I'll be assuming you know at least C. If you are okay with ASM, I recommend going with that. It will cut a lot of the hassle GBDK may generate.
+# Summary
 
-The third question to ask is the size of your project. Due to the issues with banking, I wouldn't recommend GBDK if your project is going to be large enough that banking is required, also because of the performance issue, since your project might end up being too large for the CPU, too.
+If your question is "*What should I use for my game project ?*", then you're in the right section. The first question you should ask yourself is what languages you know.
 
-If you haven't decided yet, you should consider giving ASM a try; the language is simpler than it looks. Even if you just end up writing a few functions then switching to C, you will have learned something. If you don't want to learn ASM, that's okay, too.
+### If you don't know ASM, C or C++
+Consider starting with C and GBDK. This will introduce you to working with the hardware and is an easier starting place.
 
+Once you've grasped C's concepts (most importantly pointers), give ASM a go. The language is simpler than it looks. Even if you don't manage to get working ASM code, it actually helps a lot (especially on such a constrained system) to know what's "under the hood". There is even an [online IDE](https://daid.github.io/rgbds-live/) to expiriment with.
 
-Another option is to [reach out to us](#community-and-help), and discuss the matter. 
+For C / GBDK users, knowing ASM will help you understand what it's API (which is mostly written in ASM) is doing behind the scenes and will make using emulator debuggers easier to understand.
 
+If you don't wan't to learn a language at all, GB Studio is an alternative to C and ASM.
```

Comment:
Great, we can add a link to GB Studio here, but some more context is need to highlight how distant is that approach to actually coding a game (both in terms of knowledge required and final results)

Reply:
The limitations are covered in the "GB Studio section" above- I can expand that a little more and add a hyperlink to it as is done from the "Non-Programming Language option" block.